runner.rb

```diff
@@ -2,6 +2,7 @@
 require './lib/cell'
 require './lib/board'
 require './lib/player'
+require './lib/computer'
```

Comment:
I noticed this was not included in our runner file. oddly everything worked fine before

Reply:
We could actually remove cell, board, player, and computer from this since the runner file is only interacting with the game file (which is why it was working without requiring the computer file).